README.md

```diff
@@ -26,7 +26,8 @@ Note that a model supports `/truncate_prompt` endpoint if and only if it support
 |Meta|Llama 3 Chat 8B Instruct|meta.llama3-8b-instruct-v1:0|text-to-text|🟡|🟡|❌|
 |Meta|Llama 2 Chat 70B|meta.llama2-70b-chat-v1|text-to-text|🟡|🟡|❌|
 |Meta|Llama 2 Chat 13B|meta.llama2-13b-chat-v1|text-to-text|🟡|🟡|❌|
-|Stability AI|SDXL 1.0|stability.stable-diffusion-xl-v1|text-to-image|❌|🟡|❌|
+|Stability AI|Diffusion 1.0|stability.stable-diffusion-xl-v1|text-to-image|❌|🟡|❌|
```

Comment:
I usually call the models just like they are called in the Bedrock service itself, so that a user could match one with another. Let's follow the same principle:

<img width="936" alt="Screenshot 2024-11-04 at 14 27 05" src="https://github.com/user-attachments/assets/7f5a8c3b-98ec-481e-aaa6-9c69a7fa0249">

Reply:
Fixed